fdmprinting/archive_retrieve.py

```diff
@@ -110,13 +109,13 @@ def get_status_from_db(printer_id, database):
 def save_printer_status(status, database):
     save_sql = '''
         INSERT INTO 
-        print_status(job_hash, date, printer, printer_id, state, job, mins, task_id, raw_json, image)
+        print_status(job_hash, date, printer, printer_id, state, job, mins, task_id, raw_json, image, fail_reason)
         VALUES
-        (?, CURRENT_TIMESTAMP, ?,?,?,?,?,?,?,?)
+        (?, CURRENT_TIMESTAMP, ?,?,?,?,?,?,?,?,?)
         ON CONFLICT(job_hash) 
         DO UPDATE SET 
         date = excluded.date, state = excluded.state, mins = excluded.mins, 
-        raw_json = excluded.raw_json, image = excluded.image;
+        raw_json = excluded.raw_json, image = excluded.image, fail_reason exluded.fail_reason;
```

Comment:
I think you're missing the `=` here, and you have a typo in `exluded`:

```suggestion
        raw_json = excluded.raw_json, image = excluded.image, 
        fail_reason = excluded.fail_reason;
```